instance/templates/instance/ansible/lms_users.yml

```diff
@@ -1,3 +1,4 @@
+EDXAPP_SETTINGS: 'openstack'
```

Comment:
@pomegranited Minor nit: Add empty line below this line.

Reply:
Done - I've also squashed my commits to prepare for merging.